### ISSUE TITLE
Crash report in 7.2: Can't create handler inside thread that has not called Looper.prepare()  - from AlertDialog

### ISSUE THREAD

**Issue by mzorz** [wordpress-mobile/WordPress-Android#5785]
```
Fatal Exception: java.lang.RuntimeException: Can't create handler inside thread that has not called Looper.prepare()
       at android.os.Handler.<init>(Handler.java:200)
       at android.os.Handler.<init>(Handler.java:114)
       at android.app.Dialog.<init>(Dialog.java:108)
       at android.app.Dialog.<init>(Dialog.java:148)
       at android.support.v7.app.AppCompatDialog.<init>(AppCompatDialog.java:46)
       at android.support.v7.app.AlertDialog.<init>(AlertDialog.java:97)
       at android.support.v7.app.AlertDialog$Builder.create(AlertDialog.java:929)
       at org.wordpress.android.ui.posts.EditPostActivity.hasFailedMedia(EditPostActivity.java:849)
       at org.wordpress.android.ui.posts.EditPostActivity.access$2000(EditPostActivity.java:148)
       at org.wordpress.android.ui.posts.EditPostActivity$6.run(EditPostActivity.java:1203)
       at java.lang.Thread.run(Thread.java:818)
```

**Comment by mzorz** [comment]
related to #5773

**Comment by tonyr59h** [comment]
Fixed in #5789, closing.
